jsk_pr2_robot/jsk_pr2_startup/CMakeLists.txt

```diff
@@ -20,3 +20,17 @@ endforeach()
 
 install(FILES install_pr1040_description.sh jsk_pr2.machine plugin.xml startup.app
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+
+# Create email config for PR2
+macro(configure_email_topic_yaml rname)
+  cmake_host_system_information(RESULT PR2_NAME QUERY HOSTNAME)
+  if (${PR2_NAME} MATCHES ^${rname})
+    set(RECEIVER_MAIL ${rname})
+    set(EMAIL_SENDER_ADDRESS ${PR2_NAME}@jsk.imi.i.u-tokyo.ac.jp)
```

Comment:
```suggestion
    set(EMAIL_SENDER_ADDRESS ${PR2_NAME}2@jsk.imi.i.u-tokyo.ac.jp)
```
?

Reply:
but you can pass `pr2` to `configure_email_topic_yaml`.
```
configure_email_topic_yaml(pr2)
```

Reply:
Soved with not-checking hostname